tools/xplatRegenByteCode.py

```diff
@@ -34,7 +34,7 @@ def run_sub(message, commands, error):
         ['../build.sh', '--no-jit', '--test-build', '--target-path=../out/noJit', '-j=2'], 
         'No Jit build failed - aborting bytecode generation')
     run_sub('Compiling ChakraCore with Jit',
-        ['../build.sh', '--test-build', '--target-path=../out', '-j=2'], 
+        ['../build.sh', '--test-build', '--target-path=../out/Jit', '-j=2'], 
```

Comment:
Changed the path used for this build so it's not the default path - as this build will often be a no-icu build, whereas a default build may well be one with icu - hence clashing paths = excess rebuilding.